Makefile

```diff
@@ -23,3 +23,7 @@ integration-test: build
 
 integration-test-py3: build-py3
 	docker run -v `$(HOST_TMPDIR)`:/tmp -v /var/run/docker.sock:/var/run/docker.sock docker-py3 py.test -rxs tests/integration_test.py
+
+integration-ci:
+	docker build -t dpy-tests -f ./tests/Dockerfile .
+	docker run --privileged -t dpy-tests
```

Comment:
I think one advantage is that you can use the `tests/Dockerfile` without `dind` if you want to, which can make it easier to debug problems.

It also makes it easier to see errors from the daemon (since the current setup sends them to `/dev/null`).